### ISSUE TITLE
Give separate color to Bold and Italic fonts

### ISSUE THREAD

**Issue by 0xDTC** [golam71/obsidian-hackthebox#4]
Here you can see i have marked 2 words one is bold and one is italic its merging with the same font colour where its hard to differentiate

![image](https://github.com/user-attachments/assets/358a025b-e610-450b-8eba-34d4989fd0a6)

**Comment by xp-z** [comment]
Why?

**Comment by 0xDTC** [comment]
To differentiate and highlight main stuff as the default color blend in the running notes it self for ex. if you check the screenshot you will see how it differ

![image](https://github.com/user-attachments/assets/00be0dcc-9178-4ea0-994c-ae99561ebc91)

**Comment by xp-z** [comment]
Ok, but you can highlight manually, what's the issue?

**Comment by 0xDTC** [comment]
yes i'm using my own css for that i just thought it would look better where i don't have to add my own css in the file which i'm actually using the theme css already. but yes it up to you to decide if you wanna make this change or not or you can set a poll and take action as per majority?

**Comment by i-vt** [comment]
I don't see how this feature would be useful, tbh I'd recommend to fork it if you want a custom config

**Comment by i-vt** [comment]
Hope this helps: https://help.obsidian.md/Extending+Obsidian/CSS+snippets

**Comment by 0xDTC** [comment]
No problem sir i'm already using my custom css script no worries. I'm closing this issue with this last comment🙂 working any other project lets chat on discord?
